sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/yaml/RawTask.java

```diff
@@ -76,10 +73,6 @@ public String getCmd() {
         return cmd;
     }
 
-    public String getImage() {
```

Comment:
was never used, and I don't believe it makes sense to specify an image on a task